src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -214,22 +216,28 @@ public static SupportedPlatformData GetSupportedPlatforms(CompletionItem item, W
             return null;
         }
 
-        public static int GetContextPosition(CompletionItem item)
+        public static async Task<int> GetContextPositionAsync(
+            Document document, CompletionItem item, CancellationToken cancellationToken)
         {
-            if (item.Properties.TryGetValue("ContextPosition", out var text) && int.TryParse(text, out var number))
+            if (item.Properties.TryGetValue("ContextPosition", out var text) &&
+                int.TryParse(text, out var number))
             {
-                return number;
+                // We have no access to the editor at this layer.  So it's not 
```

Comment:
Chatting with @rchande: don't we keep track of other positions for the item to avoid this sort of issue?